cmake/modules/FindSSE.cmake

```diff
@@ -92,38 +92,38 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "Android")
     CHECK_CXX_ACCEPTS_FLAG("-mavx2" _AVX2_OK)
    endif()
 elseif(CMAKE_SYSTEM_NAME MATCHES "Darwin")
-   if(NOT CPU MATCHES "arm")
-      exec_program("/usr/sbin/sysctl -n machdep.cpu.features machdep.cpu.leaf7_features" OUTPUT_VARIABLE CPUINFO)
+   if(NOT CPU MATCHES "arm" AND NOT CPU MATCHES "x86_64")
```

Comment:
is there actually a case when this is true? Isn't apple always arm or amd64?

Reply:
right, let's just remove darwin then and compilers do their thing